src/daily_output.ts

```diff
@@ -61,8 +61,9 @@ export class DailyOutput {
 	 * @param {GQLEdgeInterface[]} queryResult the edges of ArFSTransactions only - 50 block before the latest
 	 */
 	public async feedGQLData(queryResult: GQLEdgeInterface[]): Promise<void> {
-		const aggregationPromises = queryResult.map(this.aggregateData);
-		await Promise.all(aggregationPromises);
+		for (const edge of queryResult) {
+			await this.aggregateData(edge);
+		}
```

Comment:
Why the change from concurrent promises?

Reply:
Probably a leftover from debugging

Reply:
We were spamming network requests to redstone. I think that's the reason why it took longer, because by doing it sequentially we got better timings